package.json

```diff
@@ -56,7 +56,7 @@
     "@opentelemetry/api-metrics": "^0.22.0",
     "@opentelemetry/instrumentation-express": "^0.22.0",
     "@opentelemetry/instrumentation-http": "^0.22.0",
-    "@opentelemetry/instrumentation-redis": "^0.24.0",
+    "@opentelemetry/instrumentation-redis": "^0.22.0",
```

Comment:
you should use https://www.npmjs.com/package/@opentelemetry/instrumentation-ioredis if you use ioredis (i missed that :( )

Reply:
done